R/utilities.R

```diff
@@ -1320,17 +1320,14 @@ collate_results = function(sample_table,
   if(write_to_file){
     from_cache = FALSE #override default automatically for nonsense combination of options
   }
+  
   output_file = config::get("results_merged")$collated
```

Comment:
Adam after you left we discussed this further and opted to save the results to that `results/shared` folder so please change this line to 
```
output_base = config::get("project_base")
```

Reply:
In this case those both point to the same file because every gambl clone has a symlink `results` that points to `results_local`. You should familiarize yourself with the layout and organization of the gambl repo especially how we use symlinks. GAMBLR should be using the first path in this case because it's writing to `results` rather than one of the paths in the repo that is tracked in git.

Reply:
Perfect, Thanks for clarifying, makes 100% sense.